services/types.ts

```diff
@@ -34,27 +34,28 @@ export type Bed = {
 
 export type Veggie = {
   id: string;
-  veggieInfoId: string;
+  veggieInfoId?: string; // TODO: won't be optional once adding from veggies list implemented
   name: string;
-  sowDate: string;
-  harvestDate: string;
-  logs: VeggieLog[];
+  notes?: string;
+  sowDate?: string;
+  harvestDate?: string;
+  logs?: VeggieLog[];
```

Comment:
@Uken81 I have note's within each log, do you want separate notes on the plant too?